lib/pages/emergency_pass.dart

```diff
@@ -0,0 +1,142 @@
+import 'package:flutter/material.dart';
+import 'package:helpwave/services/language_model.dart';
+import 'package:helpwave/styling/constants.dart';
+import 'package:intl/intl.dart';
+import 'package:provider/provider.dart';
+import 'package:flutter_gen/gen_l10n/app_localizations.dart';
+
+class EmergencyPass extends StatefulWidget {
+  const EmergencyPass({super.key});
+
+  @override
+  State<EmergencyPass> createState() => _EmergencyPassState();
+}
+
+
+
+class _EmergencyPassState extends State<EmergencyPass> {
+  DateTime? birthDate;
+  String? organDonor;
+
+  @override
+  Widget build(BuildContext context) {
+
+    return  Consumer<LanguageModel>(
+        builder: (_, LanguageModel languageNotifier, __) {
+          return Scaffold(
+            appBar: AppBar(
+              title: Text(AppLocalizations.of(context)!.emergencyPass),),
+              body: Center(
+                  child: Padding(
+                    padding: const EdgeInsets.symmetric(horizontal: paddingMedium),
+                    child: Column(
+                      mainAxisAlignment: MainAxisAlignment.center,
+                      children: <Widget>[
+                        TextField(
+                          decoration: InputDecoration(
+                            border: const OutlineInputBorder(
+                              borderRadius: BorderRadius.all(Radius.circular(borderRadiusMedium)),
+                            ),
+                            prefixIcon: const Icon(Icons.person),
+                            labelText: AppLocalizations.of(context)!.name,
+                            hintText: AppLocalizations.of(context)!.name,
+                          ),
+                        ),
+                        Padding(
+                          padding: const EdgeInsets.symmetric(vertical: paddingSmall),
+                          child: TextField(
+                            readOnly: true,
+                            onTap: () async {
+                              DateTime? selectedDate = await showDatePicker(
+                                context: context,
+                                initialDate: DateTime.now(),
+                                firstDate: DateTime(1900),
+                                lastDate: DateTime(2100),
```

Comment:
lastdate should be datetime now

Reply:
This reminds me that we could eventually have patient data from multiple people stored on one device....

Reply:
We should discuss again